src/main/java/core/basesyntax/dao/fileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class fileReaderImpl implements fileReader {
```

Comment:
The class name `fileReaderImpl` does not follow Java naming conventions. Class names should be in PascalCase, such as `FileReaderImpl`.